docs/docs/references/roles.mdx

```diff
@@ -57,15 +57,30 @@ copy spaces/charts/dashboards from the source project they have access to.
-| Edit a Space's access (from Full to Restricted)          |      ✅      |      ✅       |      ❌       |
-| Invite users to a Restricted Space they have access to   |      ✅      |      ✅       |      ❌       |
-| Remove users from a Restricted Space they have access to |      ✅      |      ✅       |      ❌       |
-| Add/Remove content from the Space                        |      ✅      |      ✅       |      ❌       |
-| Edit the Space details (name, description, etc.)         |      ✅      |      ✅       |      ❌       |
+There are three space roles: `Full Access`, `Can Edit`, `Can View`
+
+Users with `Full Access` to a space can restrict and increase a user’s inherited project permissions in a space (e.g. you can make project editors into `can view` in a space). Space permissions determine which users can edit space contents (charts and dashboards), view the contents in a space, and change a space's settings:
+
+- A user needs to have at least the `Can view` access level to a space to see that the space exists and to see the charts and dashboards inside it.
+- A user needs to have the `Can edit` access level to a space to edit the content in the space (add/delete/rename charts and dashboards).
+- A user needs to have the `Full access` access level to a space to manage access to the space and to edit the space details (name, description, etc.).
+
+Space permissions don't otherwise control what users can do, or which data they can use to build their own content.
+
+This means:
+- a project viewer who has `Can edit` space permissions cannot get access to build or edit charts because Viewers don't have access to the Explore view
+- an interactive viewer who is given `Can edit` space permissions can save content in that space but not in any other space (unless given `edit` access to another space)
+- an editor who is given `Can view` space permissions cannot edit the content in that space, but they can create/edit content in other parts of the project (unless they're given `view` access).
+
+| Action                                                        | Full Access  | Can Edit | Can View |
+| :------------------------------------------------------------ | :----------: | :------: | :------: |
+| Edit a Space's access (from Public to Restricted)             |      ✅      |     ❌    |    ❌    |
+| Change the permissions of a user that has access to the space |      ✅      |     ❌    |    ❌    |
+| Invite users to a Restricted Space they have access to        |      ✅      |     ❌    |    ❌    |
+| Remove users from a Restricted Space they have access to      |      ✅      |     ❌    |    ❌    |
+| Edit the Space details (name, description, etc.)              |      ✅      |     ❌    |    ❌    |
+| Add/Remove content from the Space                             |      ✅      |     ✅    |    ❌    |
+| Edit content in the Space (charts, dashboards)                |      ✅      |     ✅    |    ❌    |
```

Comment:
I would invert the rows order to be similar to the org/project matrix where at the top we have more roles with access.

I would make the rows more generic, otherwise we end up with a massive table as we describe every small action: 
- View space content
- Edit space content
- Edit space and space access

Reply:
great! will update